src/zookeeper/deploy/zookeeper.yaml.template

```diff
@@ -49,6 +49,10 @@ spec:
         env:
         - name: JAVA_OPTS
           value: "-server -Xmx8192m"
+        - name: ZK_ID
```

Comment:
ZK_ID?

Reply:
It maybe not a suitable name, I fellow the config entry name `zkid`.
Do you have any suggestion?

Reply:
You have already generate the zkid through hash and I didn't find there is any place to use this value. Why shouldn't we remove it?

Reply:
Yeah, I will remove it.